store/elasticsearch/search.go

```diff
@@ -203,7 +203,35 @@ func (sr *Searcher) buildSuggestQuery(ctx context.Context, cfg discovery.SearchC
 	return payload, err
 }
 
-func (sr *Searcher) buildTextQuery(ctx context.Context, text string) elastic.Query {
+func (sr *Searcher) buildTextQuery(ctx context.Context, cfg discovery.SearchConfig) elastic.Query {
+	if len(cfg.Queries) == 0 {
+		return sr.buildTextQueryGeneric(ctx, cfg.Text)
```

Comment:
shouldn't `buildTextQuery` will always run regardless of `cfg.Queries`?
so that `cfg.Queries` will only add another filter query on top of existing query.

Reply:
Since we are using several multimatch queries and boosting urn and name field, the cfg Queries search will take a little impact. Even if we boost the cfg Queries field itself. The behaviour is not as expected.

Reply:
But if it is `filter`, that supposed to be part of cfg.Filters right?